src/test/java/com/qa/shopkick/tests/Authentication/EmailAuthentication.java

```diff
@@ -107,13 +76,9 @@ public void Test3_DeleteEmailAccount() {
 
     @Test
     public void Test3_LoginWithEmail() {
-        LoginHooks.EmailLogin();
-
-    }
+        LoginHooks.loginWithEmail();
 
-    @Test
-    public void Test4_logOutOfEmail() {
-        CustomHooks.LogMeOut();
     }
 */
+
 }
```

Comment:
TODO: Plan for the decommissioned tests to run by referring a different tag (maybe?)